setup/motor_check.cpp

```diff
@@ -43,6 +43,13 @@ void print_motor_status(const openarm::damiao_motor::Motor& motor) {
 }  // namespace
 
 int main(int argc, char* argv[]) {
+    for (int i = 1; i < argc; ++i) {
+        if (std::string(argv[i]) == "-h") {
```

Comment:
Could you use `std::string_view` to avoid needless copy?

```suggestion
        if (std::string_view(argv[i]) == "-h") {
```